cp3pt0-deployment/setup_tenant.sh

```diff
@@ -496,6 +503,10 @@ rules:
   - watch
   - deletecollection
 EOF
+    else
+        debug1 "Creating nss minimal rbac role from $MINIMAL_RBAC:\n"
+        cat ${MINIMAL_RBAC} | sed "s/operator_ns_to_replace/$OPERATOR_NS/g" > ${PREVIEW_DIR}/role.yaml
```

Comment:
I don't follow exactly, this is what is happening today ... for instance the minimal rbac rule name is `nss-managed-role-from-cloudpak-control`, that is what this replacement is doing.
```
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  name: nss-managed-role-from-operator_ns_to_replace
  namespace: ns_to_replace
```

Reply:
What I mean is, could we replace the entire name of Role entered by the user with `nss-managed-role-from-$OPERATOR_NS` to prevent them from naming the prefix `nss-managed-role-from-` incorrectly? 
I met a case today when creating a Role, the user wrote the name as `nss-managed-role-<operator-namespace>`, and there was `from` missing in the middle. I think replacing the entire name may avoid this problem, thanks